### ISSUE TITLE
Accessible Debugging

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#2671]
At some point we should find out how an accessible debug session looks like. Debug is highly interactive so I can only assume that there needs to be some feedback reporting to the user what the state of the debugged application is in. Some thoughts:
* announce when the program has started and debug session is active
* announce when it stops or a breakpoint is hit
* announce when an error is hit

I pushed a change (https://github.com/Microsoft/vscode/commit/46db05eec39a1b01eab19925a98d8307dc277f03) so that the trees in the debug viewlet (Variables, Expressions, Call Stack, Breakpoints) get announced with some meaningful data when focus is in the tree and a screen reader is connected. However, I am not sure how verbose the data should be from each of the views because the entries within can have quite some state. I suggest to check with other tools and maybe do the same.

@isidorn @weinand @stevencl @chrisdias fyi

**Comment by dstorey** [comment]
@bpasero Would be great to discuss this with @jacobrossi and @andysterland as it is something the F12 team also need figure out.

**Comment by stevencl** [comment]
We should speak to people who we are designing these experiences for to see what works for them. I will see if I can set something up.

Sharing what we learn with other teams is a good recommendation too.

**Comment by isidorn** [comment]
As Ben pointed out in a seperate issuie, we also need to make the debug hover accessibly but that should allign with our regular hover accessibility work.
